src/metabase/analytics/snowplow.clj

```diff
@@ -24,7 +29,7 @@
 
 (set! *warn-on-reflection* true)
 
-;; Adding or updating a Snowplow schema? Make sure that the two maps below are updated accordingly.
+;; Adding or updating a Snowplow schema? Make sure that the map below is updated accordingly.
```

Comment:
I don't think it's required, but to make this requirement a bit clearer, we could define `track-action!` in a similar way to this fake function:

```clojure
(mu/defn track!
  [event :- (into [:enum] (keys schema->version))]
  event)

(track! ::oops-didnt-register)
;;=> throws with message about ::oops-didnt-register not being a valid enum,
;;   and the keys from schema->version
  
```